CONTRIBUTORS.yaml

```diff
@@ -1756,6 +1756,10 @@ suchitrathapa:
     name: Suchitra Thapa
     joined: 2021-11
 
+supernord:
+    name: Johan Gustafsson
+    joined: 2024-02
+
```

Comment:
@supernord I added you in the contributor file. 
Feel free to add more information there

Reply:
like an orcid!

Reply:
Thanks @bebatut